Cargo.toml

```diff
@@ -32,10 +32,10 @@ colored = "2"
 
 
 sea-orm = { version = "0.12.4", features = [
-  "sqlx-postgres",        # `DATABASE_DRIVER` feature
-  "sqlx-sqlite",
-  "runtime-tokio-rustls",
-  "macros",
+    "sqlx-postgres", # `DATABASE_DRIVER` feature
```

Comment:
what kind of formatter do you use for `toml`?

Reply:
`RustRover` default, think it supposed to use `cargo fmt` not sure if I touched that setting.